plugins/modules/rpm_distribution.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# copyright (c) 2019, Matthias Dellweg
```

Comment:
This is a copy of ansible_distribution.py that replaces python with rpm.
Do I need to do anything different here? (and in the other modules)

Reply:
sure:
```suggestion
# copyright (c) 2020, Jacob Floyd
```
There is another author field in the `DOCUMENTATION` stanza. Can you also add your name to galaxy.yml please?